prime-router/src/main/kotlin/common/PhoneUtilities.kt

```diff
@@ -45,7 +45,7 @@ object PhoneUtilities {
      * Parses a passed in [cleanedValue] into a PhoneNumber and determines if it has an extension or not.
      * @return a boolean indicating if the value passed in is a phone number with an extension
      */
-    fun hasExtension(cleanedValue: String): Boolean {
+    fun hasPhoneNumberExtension(cleanedValue: String): Boolean {
```

Comment:
You need to also change any use of this function in the schemas as well.

Reply:
there was just one, it's fixed now